packages/time/formatter.mare

```diff
@@ -0,0 +1,517 @@
+:class TimeFormatter
+  :const month_names Array(String): [
```

Comment:
The compiler doesn't support complex types for constants yet, so this needs to be a `:fun non` for now, until I add that feature. This is the cause of the compiler bug you observed.

```suggestion
  :fun non month_names Array(String): [ // TODO: make const when array constants are supported
```

The performance will be better when it can be a `:const`, but the code should work fine as a `:fun non` for now.